internal/querycoord/query_coord.go

```diff
@@ -176,8 +178,21 @@ func (qc *QueryCoord) Init() error {
 			return
 		}
 
+		qc.chunkManager, initError = storage.NewMinioChunkManager(qc.loopCtx,
```

Comment:
Should we have something like chunkManagerFactory?
For embedded milvus I guess we don't always do storage.NewMinioChunkManager.

Also we should leave the possibility to support HDFS and other distributed storage

Reply:
It will be implemented in next path.